core/modules/system/system.install

```diff
@@ -2955,6 +2955,15 @@ function system_update_1062() {
 }
 
 /**
+ * Remove configuration settings that were never used.
```

Comment:
I don't think that this description docblock matches what this update hook does.

Reply:
It's accurate. that function deleted the user timezone settings from the date config file.

Reply:
That is what `system_update_1062()` did. We need a description for the new update, `system_update_1063()`.